### DISCUSSION TITLE
If the cache path is the same but the project path is different, the cache does not take effect

### DISCUSSION THREAD

**Discussion by zhouzhi-zz** [webpack/webpack#18517] [Q&A]
As described in the title, I currently have the same project in multiple file directories, I set cacheDirectory as the absolute path of a disk, but I can only enter the project after starting the service again to enjoy the cache, and the cache cannot take effect in other folders that have not been started. How can I make the project in multiple directories share this cache

**Comment by zhouzhi-zz** [comment]
cache: {
        type: 'filesystem',
        allowCollectingMemory: true,
        cacheDirectory: resolve('/data/gitlab-runner'),
        store: 'pack',
        memoryCacheUnaffected: true,
        name: 'prod',
        version: '1.0',   
        buildDependencies: {
            config: [__filename],
        },
    },

**Reply by alexander-akait** [reply]
Please read https://webpack.js.org/configuration/cache/#setup-cache-in-cicd-system and use https://webpack.js.org/configuration/cache/#gitlab-cicd

**Reply by zhouzhi-zz** [reply]
> Please read https://webpack.js.org/configuration/cache/#setup-cache-in-cicd-system and use https://webpack.js.org/configuration/cache/#gitlab-cicd

I work on a window, not on an ci/cd

**Reply by alexander-akait** [reply]
Why do you have `cacheDirectory: resolve('/data/gitlab-runner'),`? Also avoid using `name: 'prod'`, we use different names for develpment and production

**Reply by zhouzhi-zz** [reply]
Sorry, this is a code snippeted from a build environment. It should read `cacheDirectory: resolve("D://webpackCache"), name: "dev"`

**Reply by alexander-akait** [reply]
Avoid using `name` if you don't know why you use this option

**Reply by alexander-akait** [reply]
Look how we set the name - https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L400

**Reply by zhouzhi-zz** [reply]
I don't think it matters that I set the name, because I could have left the name field unconfigured. My current request is that I start multiple projects using the same cache. These projects are distributed under different directory folders.

**Reply by alexander-akait** [reply]
Sorry you can't do it, many things are absolute in cache, we have an issue about it, but still under dicussion

**Reply by zhouzhi-zz** [reply]
Ok, thank you for your reply. I am looking forward to having this function.
